README.md

```diff
@@ -1,2 +1,7 @@
 # hey-look-a-todo-app
-Let's make a ToDo app with @stewartmccoy because he's a real radical dude!
+A Todo app built with Django, Backbone.js, and MongoDB. Developed by [Eric Zarowny](http://www.linkedin.com/in/ezarowny/en) and [Stewart McCoy](http://www.linkedin.com/in/stewartmccoy/en).
```

Comment:
It's just running SQLite right now :P. We'll probably end up with Postgres at some point.